server/flaskr/database/__init__.py

```
@@ -28,7 +27,7 @@ def create_db_command():
 
 
 def init_db(app, uri):
-    Database.set_uri(uri)
-    
-    app.teardown_appcontext(close_db)
+    Database.init_pool(uri)
+
+    app.teardown_appcontext(lambda e: Database.close_pool())
```

Ruff (0.15.20)

[warning] Unused lambda argument: `e`

(ARG005)